### ISSUE TITLE
Git - Create a Tag Object: "type" is an enum

### ISSUE THREAD

**Issue by gr2m** [octokit/routes#99]
- endpoint scope: Git Data
- endpoint name: Create a Tag Object
- route: `POST /repos/:owner/:repo/git/tags`
- documentation URL: https://developer.github.com/v3/git/tags/#create-a-tag-object

### Current specification

https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/git/tags/create-a-tag-object.json#L37-L42

### I expected

<details>
  <summary>"type" parameter should be an enum</summary>

```json
{
  "name": "Create a Tag Object",
  "enabledForApps": true,
  "method": "POST",
  "path": "/repos/:owner/:repo/git/tags",
  "params": [
    {
      "name": "owner",
      "type": "string",
      "required": true,
      "description": ""
    },
    {
      "name": "repo",
      "type": "string",
      "required": true,
      "description": ""
    },
    {
      "name": "tag",
      "type": "string",
      "description": "The tag",
      "required": false
    },
    {
      "name": "message",
      "type": "string",
      "description": "The tag message",
      "required": false
    },
    {
      "name": "object",
      "type": "string",
      "description": "The SHA of the git object this is tagging",
      "required": false
    },
    {
      "name": "type",
      "type": "string",
      "description": "The type of the object we're tagging. Normally this is a `commit` but it can also be a `tree` or a `blob`.",
      "required": false,
      "enum": [
        "commit",
        "tree",
        "blob"
      ]
    },
    {
      "name": "tagger",
      "type": "object",
      "description": "An object with information about the individual creating the tag.",
      "required": false
    },
    {
      "name": "tagger.name",
      "type": "string",
      "description": "The name of the author of the tag",
      "required": false
    },
    {
      "name": "tagger.email",
      "type": "string",
      "description": "The email of the author of the tag",
      "required": false
    },
    {
      "name": "tagger.date",
      "type": "string",
      "description": "When this object was tagged. This is a timestamp in ISO 8601 format: `YYYY-MM-DDTHH:MM:SSZ`.",
      "required": false
    }
  ],
  "description": "Note that creating a tag object does not create the reference that makes a tag in Git. If you want to create an annotated tag in Git, you have to do this call to create the tag object, and then [create](/v3/git/refs/#create-a-reference) the `refs/tags/[tag]` reference. If you want to create a lightweight tag, you only have to [create](/v3/git/refs/#create-a-reference) the tag reference - this call would be unnecessary.",
  "documentationUrl": "https://developer.github.com/v3/git/tags/#create-a-tag-object"
}
```
</details>

---

If you’d like to work on it, just comment below and follow instructions in [CONTRIBUTING.md](https://github.com/octokit/routes/blob/master/CONTRIBUTING.md)

**Comment by knrt10** [comment]
I would like to work on it.

**Comment by gr2m** [comment]
fixed via #105
